NumLib/DOF/DOFTableUtil.cpp

```diff
@@ -22,8 +47,7 @@ double getNodalValue(GlobalVector const& x, MeshLib::Mesh const& mesh,
 
     auto const index = dof_table.getLocalIndex(
         l, global_component_id, x.getRangeBegin(), x.getRangeEnd());
-    if (index == NumLib::MeshComponentMap::nop)
-        return 0.0;
+    assert (index != NumLib::MeshComponentMap::nop);
```

Comment:
Is this assertion necessary only to check that the code of OGS is consistent or is it there to check if user-provided input data is correct? In the latter case the check should also be there in release mode.
Apart from that this commit looks good.